arch/arm/core/mpu/arm_core_mpu.c

```diff
@@ -106,6 +111,18 @@ static const struct z_arm_mpu_partition static_regions[] = {
 #endif
 	},
 #endif /* CONFIG_CODE_DATA_RELOCATION_SRAM */
+#if defined(CONFIG_SRAM_VECTOR_TABLE)
+	{
+		/* Vector table in SRAM */
+		.start = (uint32_t)&_sram_vector_start,
+		.size = (uint32_t)&_sram_vector_size,
+#if defined(CONFIG_ARM_MPU_PXN) && defined(CONFIG_USERSPACE)
+		.attr = K_MEM_PARTITION_P_R_U_RX,
+#else
+		.attr = K_MEM_PARTITION_P_RX_U_RX,
```

Comment:
I believe `_sram_vector_start` only contains data. So, maybe `K_MEM_PARTITION_P_RO_U_RO` is sufficient? (unless it is strategy to coalesce the MPU segments)